docker/nginx.conf

```diff
@@ -41,6 +41,16 @@ http {
             add_header Pragma "no-cache";
         }
 
+        location /api/event-stream {
+            rewrite /api/(.*) /$1 break;
+            proxy_pass <search-store-root>;
+            proxy_http_version 1.1;
+            proxy_set_header Upgrade $http_upgrade;
```

Comment:
Do we really need this?

Reply:
> [...] WebSocket is a hop‑by‑hop protocol, so when a proxy server intercepts an Upgrade request from a client it needs to send its own Upgrade request to the backend server [...]
https://www.nginx.com/blog/websocket-nginx/

I'm searching for more sources for that.